META-INF/com/google/android/aroma/install_custom.edify

```diff
@@ -57,12 +57,11 @@ endif;
 #KERNEL
 ###############################################
 menubox(
-    "Root Solution",
+    "Kernel Option",
     "Select your Kernel type, installs Eureka kernel and DTB 5",
-    "@su", 
+    "@kern", 
```

Comment:
there is no icon named kern. why changing it ?

Reply:
what are you talking about? i didnt change it.